docker/federation-test/servers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+sudo rm -rf volumes
+
+pushd ../../server/
+cargo build
+popd
+
+pushd ../../ui
+yarn
+yarn build
+popd
+
+mkdir -p volumes/pictrs_{alpha,beta,gamma}
+sudo chown -R 991:991 volumes/pictrs_{alpha,beta,gamma}
+
+sudo docker build ../../ --file ../federation/Dockerfile --tag lemmy-federation:latest
+
+sudo docker-compose --file ../federation/docker-compose.yml --project-directory . up
```

Comment:
Isnt this effectively thie same as `docker/federation/run-federation-test.bash`?

Reply:
I created servers.sh and tests.sh so I could run them in different windows and see logs